repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthFilter.scala

```diff
@@ -100,19 +98,34 @@ class RackspaceIdentityBasicAuthFilter @Inject()(configurationService: Configura
   }
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain) {
-    new FilterLogicHandlerDelegate(servletRequest, servletResponse, filterChain).doFilter(this)
+    val wrappedHttpRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
+    val httpResponse = servletResponse.asInstanceOf[HttpServletResponse]
+
+    val filterAction = handleRequest(wrappedHttpRequest, httpResponse)
+    filterAction match {
```

Comment:
The original `FilterAction` in `org.openrepose.core.filter.logic` will be going away with the `FilterDirector`. The new one I added to `org.openrepose.commons.utils.servlet.filter` allows us to do this transition fairly cleanly. The original behavior in `FilterDirector` for the `NOT_SET` action was to throw a 500 if no filter had changed the value. I left it in the new one strictly for ease of conversion, but I agree that it should probably be tossed. The remaining three options:

* `PASS`
* `PROCESS_RESPONSE`
* `RETURN`

provide a way to route the logic internal to the filter if needed. I'm have no strong opinion either way on using the new `FilterAction` moving forward or just using local definitions other than to say it helps make them consistent if we use a common convention. What that convention is, makes no difference to me. I would just like to be able to look at a new to me filter and be able to easily follow the flow. To me, that was the one nice thing the `FilterAction` provided and that is why I brought it along.